README.md

```diff
@@ -48,6 +48,21 @@ path = Path.expand("~/workspace/elixir-docker-guide")
   ExDockerBuild.DockerBuild.build(path)
 ```
 
+Or you can start using escript:
+
+```ex
+mix escript.build
+Generated escript ex_docker_build 
+```
+
+Then call the escript passing the path to a Dockerfile
+
+```ex
+./ex_docker_build ~/workspace/elixir-docker-guide
```

Comment:
you need to pass the path to a file, not to a dir, i think you forgot to add `/Dockerfile`